external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/FieldConstants.java

```diff
@@ -23,5 +23,6 @@
   public static final String Offset = "offset";
   public static final String Message = "message";
   public static final String META_DATA = "metadata";
+  public static final String SYSTEM_META_DATA = "systemmetadata";

```

Comment:
>SYSTEM_META_DATA [](start = 29, length = 16)

Too generic; in general being more specific is recommended - feel free to use something like eventdata_system_properties